### PR TITLE
Fix for Issue 37 & 38. Modified move function to check deckType (now in Deck class)

### PR DESCRIPTION
**Pull request by terri-hewitt**
…s Spanish.

### CODE REVIEW

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
     public int score;
 
+    public int SpanishDeck; //1 = deck type is Spanish, 0 = deck type is English
```
Comment (elitucker) [inline src/main/java/models/Game.java]: I think this might be better under the Deck class. It seems to fall under the responsibility of that class.

Comment (terri-hewitt) [reply src/main/java/models/Game.java]: Good point, I'll move it to the Deck class, and submit a new pull request with the changes I made for Issue #38 and Issue #37.

**Comment by terri-hewitt** [comment]
Closed to create new branch with changes for issues #38 and #37.

**Comment by terri-hewitt** [comment]
Ok, I moved the deckType variable into the Deck class and also made changes to the remove function to perform additional checks/actions if the deckType is set to 1 (Spanish).

**Comment by Lucas-Campos-Davis** [comment]
Should we merge this before or after merging the Spanish deck?

**Comment by terri-hewitt** [comment]
Let's merge the Spanish deck first and see if there are any merge conflicts.

**Pull request closed without merging**